Scripts/run_tests.sh

```diff
@@ -16,7 +16,7 @@ cd "$PROJECT_ROOT_PATH"
 
 function xcodebuild_test() {
     xcodebuild test \
-        -project "Sources/$XCODE_PROJECT" \
+        -project "$XCODE_PROJECT" \
```

Comment:
We should also update `SWIFT_FORMAT_INPUT_FILENAMES` in `lint_code.sh` and `format_code.sh` to include the newly created `./Tests` directory.